requirements.txt

```
@@ -3,7 +3,7 @@ cachetools~=4.1
 click<8.0
 cookiecutter~=1.7.0
 dynaconf>=3.1.2,<4.0.0
-fsspec>=2021.04, <2022.01  # Upper bound set arbitrarily, to be reassessed in early 2022
+fsspec>=2021.04, <=2022.1
```

```suggestion
fsspec>=2021.4, <=2022.1
```
`fsspec` releases don't have leading 0 on the CalVer month.

---

Good catch, thank you!!